js/ad_manager_vast.js

```diff
@@ -369,16 +369,24 @@ OO.Ads.manager(function(_, $) {
 
       // Save the stream data for use by VideoController
       var streams = {};
+      var streamData = null;
       var linearAd = ad.data.linear;
       if (linearAd && linearAd.mediaFiles) {
         var vastStreams = linearAd.mediaFiles;
-        streams["webm"] = this._extractStreamForType(linearAd.mediaFiles, "webm");
-        if (ad.streamUrl == null) {
-          streams["mp4"] = this._extractStreamForType(linearAd.mediaFiles, "mp4");
-          OO.log("extract m3u8 stream here"); // TODO
+        var videoEncodingsSupported = OO.VIDEO.ENCODING;
+        for (var encoding in videoEncodingsSupported) {
+          streamData = null;
+          if (videoEncodingsSupported.hasOwnProperty(encoding)) {
```

Comment:
What situation does this protect from?  The encoding property is coming from the same variable.